fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/BadgeTokens.kt

```diff
@@ -72,4 +74,7 @@ open class BadgeTokens : IControlToken, Parcelable {
             FluentTheme.aliasTokens.neutralStrokeColor[FluentAliasTokens.NeutralStrokeColorTokens.StrokeFocus1].value()
         )
     }
+
+    @Composable
+    open fun cornerRadius(badgeInfo: BadgeInfo): Dp = 100.dp
```

Comment:
Can we use the CornerRadius Token for circle here instead?

Reply:
Ideally, we could but for now using the values which is mentioned in Figma. I have added the comment asking same question.